### PR TITLE
ENH: Add task/action to automate periodic download of images

### LINKED ISSUE #67: ENH: Add task/action to automate periodic download of images
- in cropclassification.py -> add action "calc_periodic_images"
- add script calc_periodic_images.py in "/cropclassification", based on "/bin_util/calc_periodic_images.py"
- add section "calc_periodic_images" to general.ini and config_helper.py with parameters needed for "calc_periodic_images.py"
    - remark: to configure end date, provide following options:
        - `end_date`: also supports placeholder {now}, to use the current date
        - `end_date_subtract_days`: if not None, subtract `end_date_subtract_days` days from `end_date`
- test using e.g. `roi_bounds=[161_000, 188_000, 162_000, 189_000],` and 1 week of data

### PR DESCRIPTION
**Pull request by KriWay**
- add task and test periodic download

resolves #67

### CODE REVIEW

**Review by theroggy — CHANGES_REQUESTED** [review]
Your `if not simulate:` check made me wonder, maybe this principle can be used to make to move the script to make it a real test anyway?

File: cropclassification/general.ini [hunk 1]
```diff
@@ -39,6 +39,28 @@ reuse_last_run_dir = False
...
+# If not None, subtract end_date_subtract_days from end_date
+end_date_subtract_days = 3
+# The path to download the mosaics
+dest_image_data_dir = /tmp/satellite_periodic
+# The sensor
+sensors = s2-agri
+# The CRS of the roi
+roi_crs = 31370
+# Simulate, nothing is being downloaded
+simulate = MUST_OVERRIDE
```
Comment (theroggy) [inline cropclassification/general.ini:62]:
```suggestion
simulate = False
```
I think it is more logical to use False as default here?

It is typically only for testing purposes that simulate is relevant?

**Review by theroggy — CHANGES_REQUESTED** [review]
Looks great.

Just one small remark...

File: cropclassification/general.ini (hunk 1 above)
Comment (KriWay) [reply cropclassification/general.ini:62]: yes indeed

**Review by theroggy — APPROVED** [review]
Nice, ready to go!

**Pull request merged**